cdap-standalone/src/test/java/co/cask/cdap/StandaloneTester.java

```diff
@@ -59,7 +59,6 @@ protected void before() throws Throwable {
     cConf.set(Constants.Router.ADDRESS, getLocalHostname());
     cConf.setInt(Constants.Router.ROUTER_PORT, Networks.getRandomPort());
     cConf.setBoolean(Constants.Dangerous.UNRECOVERABLE_RESET, true);
-    cConf.setBoolean(Constants.Explore.EXPLORE_ENABLED, true);
```

Comment:
If there are any tests that use standalone but don't need explore, we should set explore.enabled to `false` for those. Otherwise, the tests will take a lot longer to run.